content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md

```diff
@@ -77,22 +77,24 @@ The architecture components, from right to left in the figure, consist of the fo
 * [Metrics API](#metrics-api): Kubernetes API supporting access to CPU and memory used for
   workload autoscaling. To make this work in your cluster, you need an API extension server that
   provides the Metrics API.
-  
+
   {{< note >}}
   cAdvisor supports reading metrics from cgroups, which works with typical container runtimes on Linux.
   If you use a container runtime that uses another resource isolation mechanism, for example
   virtualization, then that container runtime must support
   [CRI Container Metrics](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/cri-container-stats.md)
   in order for metrics to be available to the kubelet.
   {{< /note >}}
-  
+
 <!-- body -->
 
 ## Metrics API
 
```

Comment:
You may want to add something like the following here:

```
{{< feature-state for_k8s_version="1.24" state="beta" >}}
```

Reply:
I'm not sure which Kubernetes release promoted `metrics.kubernetes.io` to beta - it was a while ago, right?

Reply:
Right. Just checked this: https://github.com/kubernetes/metrics/commit/5c7f1617ad5e9ceada708f5be3f93918d7b724a4

It was a loooooong time ago.

Reply:
Yes the graduation to GA is long due, we have a KEP for it now at least: https://github.com/kubernetes/enhancements/issues/3048